app/controllers/objects_controller.rb

```diff
@@ -72,13 +72,9 @@ def csv_checksum_list
   end
 
   def checksum_for_object(druid)
```

Comment:
one could argue this method should go in MoabStorageService - I'm open to that.

Reply:
meh